pkg/sql/crdb_internal_test.go

```diff
@@ -517,6 +518,7 @@ UPDATE system.namespace SET id = 12345 WHERE id = %d;
 func TestDistSQLFlowsVirtualTables(t *testing.T) {
 	defer leaktest.AfterTest(t)()
 	defer log.Scope(t).Close(t)
+	skip.UnderRace(t)
```

Comment:
Why only under race? I think we want "skip.WithIssue(t, 75208)". PS if you're lazy:

```
make bin/skip-test; bin/skip-test --issue_num=#75208 pkg/sql:TestDistSQLFlowsVirtualTables
```